Code/client/Services/Generic/MagicService.cpp

```diff
@@ -314,15 +314,15 @@ void MagicService::OnAddTargetEvent(const AddTargetEvent& acEvent) noexcept
     if (it == std::end(view))
     {
         spdlog::warn("Form id not found for magic add target, form id: {:X}", acEvent.TargetID);
-        m_queuedEffects[acEvent.TargetID] = request;
+        m_queuedEffects[acEvent.TargetID] = std::pair<int, AddTargetRequest>(0, request);
         return;
     }
 
     std::optional<uint32_t> serverIdRes = Utils::GetServerId(*it);
     if (!serverIdRes.has_value())
     {
         spdlog::warn("Server id not found for magic add target, form id: {:X}", acEvent.TargetID);
-        m_queuedEffects[acEvent.TargetID] = request;
+        m_queuedEffects[acEvent.TargetID] = std::pair<int, AddTargetRequest>(0, request);
```

Comment:
make_pair pls

Reply:
It's even simpler since C++17. But I refuse to use the "simpler" but less readable initializer syntax, '= {0, request};'